docs/src/developing/clients/jsonrpc-api.md

```diff
@@ -120,8 +120,8 @@ Requests can be sent in batches by sending an array of JSON-RPC request objects
 
 - Hash: A SHA-256 hash of a chunk of data.
 - Pubkey: The public key of a Ed25519 key-pair.
-- Signature: An Ed25519 signature of a chunk of data.
-- Transaction: A Solana instruction signed by a client key-pair.
+- Transaction: A list of Solana instructions signed by a client key-pair to authorize those actions.
```

Comment:
```suggestion
- Transaction: A list of Solana instructions signed by a client keypair to authorize those actions.
```
Nit: I know this predates your changes, but this is the only place we hyphenate keypair...

Reply:
oh, consistency outweighs. :)